ui/scss/core/sim_ui/_main.scss

```diff
@@ -1,42 +1,64 @@
 .sim-content {
-  min-width: calc(275px + 1vw);
-  display: flex;
-  flex-flow: column;
-
-  .sim-main {
-    height: 80%;
-    display: flex;
-    flex-grow: 1;
-
-    .tab-pane {
-      max-width: 100%;
-      flex-grow: 1;
-      padding-top: $gap-width;
-      padding-bottom: $gap-width;
-    }
-  }
+	min-width: calc(275px + 1vw);
+	display: flex;
+	flex-flow: column;
+
+	.sim-main {
+		height: 80%;
+		display: flex;
+		flex-grow: 1;
+
+		.tab-pane {
+			max-width: 100%;
+			flex-grow: 1;
+			padding-top: $gap-width;
+			padding-bottom: $gap-width;
+		}
+	}
 }
 
 .sim-crash-report {
-  .sim-crash-report-header {
-    font-size: 1rem;
-    font-family: SimDefaultFont;
-  }
+	.sim-crash-report-header {
+		font-size: 1rem;
+		font-family: SimDefaultFont;
+	}
 
-  .sim-crash-report-text {
+	.sim-crash-report-text {
 		width: 100%;
 		height: 80vh;
 		resize: none;
-  }
+	}
+}
+
+@include media-breakpoint-down(lg) {
+	.sim-content {
+		.sim-main {
+			.tab-pane {
+				padding-top: $gap-width-sm * 2;
+				padding-bottom: $gap-width-sm * 2;
+			}
+		}
+	}
+}
+
+@include media-breakpoint-down(xxl) {
+	.sim-sidebar {
+		.sim-sidebar-content {
+			padding-left: $gap-width-sm * 2;
+			padding-right: $gap-width-sm * 2;
+
+			.sim-sidebar-actions {
+				padding: 0;
+				margin: 0;
+			}
+		}
+	}
 }
 
 @include media-breakpoint-down(lg) {
-  .sim-content {
-    .sim-main {
-      .tab-pane {
-        padding-top: $gap-width-sm * 2;
-        padding-bottom: $gap-width-sm * 2;
-      }
-    }
-  }
+	.sim-sidebar {
+		.sim-sidebar-content {
+			padding: $gap-width-sm * 2 $gap-width-sm;
+		}
+	}
```

Comment:
I feel like at some point we need to just blow away a bunch of CSS and rewrite it 🙃